.NET/Microsoft.Recognizers.Text.Number/Extractors/BaseNumberRangeExtractor.cs

```diff
@@ -210,6 +222,31 @@ private List<ExtractResult> ExtractNumberAndOrdinalFromStr(string numberStr)
             extractNumber = extractNumber.OrderByDescending(num => num.Length).ThenByDescending(num => num.Start).ToList();
             return extractNumber;
         }
+
+        // Fractions like "2 in 2009", "in" is ambiguous connector in some of the context
+        // This function is to process and remain only the part before ambiguous connector
+        private List<ExtractResult> ProcessAmbiguousFractions(List<ExtractResult> ers)
```

Comment:
This could should probably go inside ExtractNumberAndOrdinalFromStr

Reply:
Fixed.